### ISSUE TITLE
Set static IP from UI

### ISSUE THREAD

**Issue by Mike710Shine** [IceWhaleTech/CasaOS#1175]
I would be nice if we can change the IP Adress from dhcp to static IP-Address and back over the CasaOS WebUI.

**Comment by ETWang1991** [comment]
got it

**Comment by Hedius** [comment]
One thing. there are different methods for managing IPs (network-manager, manual, systemd-networkd. etc.)
And depending on the operating system this can create conflicts.

**Comment by Mike710Shine** [comment]
Yes that's right but isn't there a way that the person who wants to set it can select the substructure and then configure the IP?

**Comment by Draxter** [comment]
This would be risky, as setting a wrong subnet would make the server unreachable, might be wise for it to have some sanity check.

Allowing a host name change along with it would be good too.
